Resources/DataTransformer.swift

```
@@ -0,0 +1,47 @@
+//
+//  LocalStorage.swift
+//  Pigeon
+//
+//  Created by Kara Graysen on 2021-06-30.
+//  Copyright © 2021 9436-7265 Quebec Inc dba Pigeon Documents. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+struct DataTransformer {
+    static func transform(document: Document) -> String {
+        let jsonEncoder = JSONEncoder()
+        
+        var jsonData: Data? = nil
+        do {
+            jsonData = try jsonEncoder.encode(document)
+        } catch let error {
+            print(error.localizedDescription)
+        }
+        
+        if let unwrappedJsonData = jsonData, let jsonString = String(data: unwrappedJsonData, encoding: String.Encoding.utf16) {
+            return jsonString
+        }
+        
+        return "null"
```

Use `throws` instead of just returning a string with null in it.

---

idk how to do this so I want to do this later but for now it's fine since the code is being replaced.